test/evergreen.yml

```diff
@@ -1783,6 +1808,32 @@ tasks:
             set -o verbose
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
+  
+  - name: format-stress-sanitizer-test
+    #set a 25 hours (25*60*60 = 90000 seconds) timeout
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger address sanitizer"
+      - func: "format test script"
+        vars:
+          test_env_vars: ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer
+          # run for 24 hours ( 24 * 60 = 1440 minutes), don't stop at failed tests, use default config
+          format_test_script_args: -t 1440
+  
+  - name: format-stress-sanitizer-smoke-test
+    #set a 25 hours timeout
+    exec_timeout_secs: 90000
```

Comment:
This setting feels a bit over the needed. Will the default setting (6 hrs) be enough for this task?

Reply:
This one takes about 5h30mins to finish so 6hr limit may or may not be an issue. Perhaps 7hr might be good number.

Reply:
7 hrs sounds good to me.